spec/services/publishers/vacancies/vacancy_step_process_spec.rb

```diff
@@ -144,22 +138,6 @@
             expect(subject.steps).not_to include(:education_phases)
           end
         end
-
-        context "when the schools have different phases" do
-          let(:school2) { create(:school, :primary) }
-
-          it "includes the `education_phases` step" do
-            expect(subject.steps).to include(:education_phases)
-          end
-
-          context "when the vacancy has a phase already" do
-            let(:vacancy) { create(:vacancy, phase: "secondary", organisations: [school, school2]) }
-
-            it "still includes the `education_phases` step" do
-              expect(subject.steps).to include(:education_phases)
-            end
-          end
-        end
```

Comment:
So it seems you've removed all the tests that check to see if education_phase is in the steps. Don't we still need to test for this? It's dependant on `allow_phase_to_be_set?` ,right?

Shouldn't we have two contexts here, one for when `allow_phase_to_be_set?` is true and one for when it's false? 

Then we test `allow_phase_to_be_set?` in the vacancy model spec?